2016/ideas-list-biocore.md

```diff
@@ -1,9 +1,63 @@
 # biocore's Idea Page
 
+To be clear, your own ideas are most welcome!
+
 ## A general purpose next-generation sequence pre-processor for QIIME 2
 
 Various quality control and other sequence "pre-processing" steps are required of microbiome "next-generation" sequencing data before it can used in a QIIME analysis. We propose the development of a general purpose sequence pre-processor that would be developed as a [QIIME 2 plugin](https://github.com/biocore/qiime2). This is described in detail in an issue [here](https://github.com/biocore/qiime/issues/1954).
```

Comment:
Could you copy some of the information from the issue to here? Google doesn't like very much issues for GSoC.